styles.css

```diff
@@ -0,0 +1,59 @@
+*{
+    padding: 0;
+    margin: 0;
+    background-image: linear-gradient(to bottom right, black, rgb(42, 0, 81)); 
+}
+#header{
+    text-align: center;
+    background-color: rgb(70, 5, 131);
+    padding: 50px;
+    color: aliceblue;
+    font-weight: bolder;
+    font-size: 40px;
+    background-image: none;
+
+}
+#generate-btn{
+    margin: 5rem 25rem 0rem 36rem;
+    text-align: center;
+    font-size: 20px;
+    padding: 10px 20px;
+    border-radius: 4px;
+    background-color: aliceblue;
+    color: black;
+    border: 2px solid #371F76;
+    background-image: none;
+}
+.quote-container{
+    background-image: none;
```

Comment:
why is background-image: none; used in all classes?
don't repeat CSS, use inheritance if possible - read

Reply:
i did not want to override the same property everywhere, will use inheritance.